components/scrollable-container/script.js

```diff
@@ -42,13 +42,16 @@ class ScrollableContainer extends BaseComponent {
     set scrollPos(value) {
         this._scrollPos = value;
         // set the position of the scrollbar handle
-        this.scrollbar.handlePosition = this._scrollPos;
+        components.waitForFrames(() => this.scrollbar.scrollToPercents(this._scrollPos));
     }
 
     // eslint-disable-next-line require-jsdoc
     attributeChangedCallback(name, oldValue, newValue) {
+        if (!this.isRendered) return;
+
         // boolean attributes' initial value is an empty string, so
         // we need to check if the old value was null
+        if (name === 'fixed-slider-height') return this.shouldShowScrollbar();
```

Comment:
Maybe saved `fixed-slider-height` to a constant, in case we need to change it in the future?

Reply:
Fixed